### DISCUSSION TITLE
Customize messages

### DISCUSSION THREAD

**Discussion by Delvio** [codingjoe/django-select2#299] [Q&A]
If I wanted to customize the messages that we get when the input is too short or when there are no results found, how do you do it?

<img width="513" alt="Screenshot 2024-09-15 at 8 59 16 PM" src="https://github.com/user-attachments/assets/cafe0a69-2a2b-4c90-80b4-2b63fb152267">
<img width="565" alt="Screenshot 2024-09-15 at 8 59 12 PM" src="https://github.com/user-attachments/assets/c81e18e3-bb69-4f3f-b473-416a3b16c9f5">

**Comment by codingjoe** [comment]
Hi there 👋,

I recommend to reach out to the Select2 community. We only handle the Django part here. But I remember we had similar questions here asked before, https://github.com/codingjoe/django-select2/issues/217

Maybe those are helpful.

Cheers!
Joe
